pages/join/[hash].js

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+import * as api from '../../api';
+import { useRouter } from 'next/router';
+
+function Join({ gameHash, playerHash }) {
+    const router = useRouter();
+
+    useEffect(() => {
+        if (gameHash && playerHash) {
+            setTimeout(function () {
```

Comment:
It's a timeout to let the user read the screen?

Reply:
Yes